configure.ac

```diff
@@ -74,14 +74,11 @@ AS_IF([test "x$enable_gtk3" = xyes],
 AM_CONDITIONAL([GTK3], [test "x$gtk_package" = "xgtk+-3.0"])
 
 # GTK/GLib/GIO checks
-gtk_modules="$gtk_package >= $gtk_min_version glib-2.0 >= 2.28"
-gtk_modules_private="gio-2.0 >= 2.28 gmodule-no-export-2.0"
+gtk_modules="$gtk_package >= $gtk_min_version glib-2.0 >= 2.32"
+gtk_modules_private="gio-2.0 >= 2.32 gmodule-no-export-2.0"
```

Comment:
It's more subtle than that because this commit set also uses the generic marshaller new in 2.30.  We could generate our own though.

But anyway, we'll probably want upgrading GLib at some point for other reasons, so it might make sense to find a way to have it on Windows GTK2.  It's a fair bit annoying though, as the GTK2 bundle used to be basically all we really needed, and that wouldn't be the same anymore.

Though, it might be fairly easy to pick up GLib only from http://win32builder.gnome.org/ and use that one (2.34.3), if it doesn't requires fetching too much extra.  Or merge the GTK2 and GTK3 bundles and rely on GTK versions not having any name clashes.